gdc_client/download/client.py

```diff
@@ -65,12 +65,19 @@ def download_annotations(self, file_id):
         annotation_list = ','.join(annotations)
 
         if annotations:
-            log.debug('Found {0} annotations for {1}.'.format(
-                len(annotations), file_id))
+            log.debug('Found {0} annotations for {1}.'.format(len(annotations), file_id))
+            # TODO: Refactor this with the _post method for a generic function for calling active/legacy api
             r = requests.get(
                 urlparse.urljoin(self.data_uri, annotation_list),
                 params={'compress': True},
                 verify=self.verify)
+            if r.status_code != requests.codes.ok:
+                # try legacy if active doesn't return OK
+                r = requests.get(
+                    urlparse.urljoin(self.legacy_data_uri, annotation_list),
```

Comment:
This might go over the max character limit of a URL. You can hit this endpoint with a POST request and give it a body containing the IDs.